app.js

```diff
@@ -0,0 +1,319 @@
+"use strict"
+
+var beers = [];
+
+function Beer(name, img, type, flavor, hoppy, store, address, lat, long) {
+  this.name = name;
+  this.img = img;
+  this.type = type;
+  this.flavor = flavor;
+  this.hoppy = hoppy;
+  this.store = store;
+  this.address = address;
+  this.lat = lat;
+  this.long = long;
+}
+
+(function() {
+
+  $.ajax({
+    url: 'beer_file.json',
+    async: false,
+    dataType: 'json',
+    success: function(data) {
+      for(var x in data) {
+        beers[x] = new Beer(data[x].name, data[x].img.toLowerCase(), data[x].type.toLowerCase(), data[x].flavor.toLowerCase(), data[x].hoppy.toLowerCase(), data[x].store, data[x].address, data[x].lat, data[x].long);
+      }
+    }
+  });
+
+}) ()
+
+var controller = {
+  userType: '',
+  userTypePic: '',
+  userFlavor: '',
+  userFlavorPic: '',
+  userHoppy: '',
+  userHoppyPic: '',
+  inputCount: 0,
+
+
+  checkCount: function() {
+    if(this.inputCount === 3) {
+      document.getElementById('beer_button').style.visibility = 'visible';
+    }
+  },
+
+  drawBeerType: function() {
+
+    var type = document.getElementById('type');
+    var parent = type.parentNode;
+    parent.removeChild(type);
+    var newUl = document.createElement('ul')
+    newUl.setAttribute('id', 'newTypeUl');
```

Comment:
Just a note that `newUl` is a JS object, so you can also access these properties/attributes by doing `newUl.id = ...`